cx-sig-charter-draft.md

```diff
@@ -0,0 +1,81 @@
+  - Contributor Summit(s) 
+  - [Gardening Days and hackathons](https://github.com/spinnaker-hackathon/gardening)
+  - Retrospective moderation for other SIGs upon request
+  - Other events, like other SIG face-to-face events, upon request and consideration
+- Strategize, build, and execute on scalable mentoring programs for all contributor levels. These may include:
+  - [Google Summer of Code](https://summerofcode.withgoogle.com/archive/)
+  - [Outreachy](https://www.outreachy.org/)
+- Mentoring programs, such as: office hours, meet the contributors, 1:1 pairing, group mentoring, etc.
+- Help onboard new and current contributors into the culture, workflow, and CI of our contributor experience through the [contributor guide](https://spinnaker.io/community/contributing/), other related documentation, contributor summits, and programs tailored to onboarding.
+- Perform issue triage on and maintain the spinnaker/contributor-experience repository.  
+- Help SIGs with being as transparent and open as possible through creating best practices, guidelines, and general administration of YouTube, Zoom, and our mailing lists where applicable
+- Document and assist SIGs/WG Chairs and Technical Leads with organizational management operations
+- Distribute contributor related news on various Spinnaker channels, including Continous Delivery Foundation ([CDF](https://cd.foundation)) for posting blogs, social media, and other platforms as needed.
+- Establish and share metrics to measure project health, community health, and general trends, including:
+  - [DevStats](https://spinnaker.devstats.cd.foundation/)
+  - the CDF contribution gamification app [[WIP](https://github.com/ExitoLab/spinnaker_gamification_app)]
+  - contributor experience survey(s)
+  - engagement on project platforms
+  - pull request and issue engagement and closure times
+- Set community health goals such as improved PR and issue engagement and closure times, and track progress towards their achievement using established metrics.
+- Research other OSS projects and consult with their leaders about contributor experience topics to make sure we are always delivering value to our contributors
+- Create and maintain subprojects that incentivize community outreach, individual content creation, and welcoming/onboarding of new contributors
+
+#### Cross-project and Externally Facing Processes
+
+We cross-cut all SIGs and working groups to deliver the following processes:
+
+- Deploying Changes:
+ This SIG will create change-deployment policies that balance quick response to the needs of the community and a limit on disruptions for project contributors. We will create policies to socialize and elicit feedback on changes via community channels such as Slack, spinnaker.io, Twitter, mailing lists, SIG meetings, delegating to SIG leads, etc. The amount of notice we provide and the amount of consensus we seek is driven by our estimation of risk. We can estimate risk based on these parameters:
```

Comment:
Does this mean that this SIG is empowered to establish policies about how all SIGs conduct their business?  (For example, requiring a particular platform for meetings, requiring a particular meeting cadence, etc.).

Similar to the above question, I'm wondering how much oversight/approval we'd expect the TOC/SC to provide in these cases.

Reply:
Updates have provided clarify/resolved this.